modules/03-network-resiliency/rate-limiting/README.md

```diff
@@ -22,13 +22,12 @@ Looking into the contents of the file [local-ratelimit.yaml](local-ratelimit/loc
 
 ### Test
 
-To test the rate limiter in action, exec into a pod in the mesh, in our example 
-below it is the `catalogdetail` pod and send a bunch of requests to the `prodcatalog` service to trigger the rate limiter. 
+To test the rate limiter in action, exec into a pod in the mesh, in our example below it is the `frontend` pod and send a bunch of requests to the `prodcatalog` service to trigger the rate limiter. 
```

Comment:
Better to state "To test the rate limiter in action, exec into a `frontend` pod in the mesh"